spec/rubocop/cop/factory_bot/id_sequence_spec.rb

```diff
@@ -103,4 +103,20 @@
       end
     RUBY
   end
+
+  it 'does not register an offense for a `sequence` with non-symbol argment' do
+    expect_no_offenses(<<~RUBY)
+      FactoryBot.define do
+        sequence(id)
+      end
+    RUBY
+  end
+
+  it 'does not register an offense for a `sequence` without argument' do
+    expect_no_offenses(<<~RUBY)
+      FactoryBot.define do
+        foo.sequence
```

Comment:
This is no argument, but also an explicit receiver call. Is this intentional ?

Reply:
Indeed it is. I updated it.